src/test/fuzz/deserialize.cpp

```diff
@@ -34,7 +40,133 @@ void test_one_input(const std::vector<uint8_t>& buffer)
         return;
     }
 
-#if BLOCK_DESERIALIZE
+#if BLOCK_FILTER_DESERIALIZE
+    try {
+        BlockFilter block_filter;
+        ds >> block_filter;
+    } catch (const std::ios_base::failure&) {
+    }
+#elif ADDR_INFO_DESERIALIZE
+    try {
+        CAddrInfo addr_info;
+        ds >> addr_info;
+    } catch (const std::ios_base::failure&) {
+    }
+#elif BLOCK_FILE_INFO_DESERIALIZE
+    try {
+        CBlockFileInfo block_file_info;
+        ds >> block_file_info;
+    } catch (const std::ios_base::failure&) {
+    }
+#elif BLOCK_HEADER_AND_SHORT_TXIDS_DESERIALIZE
+    try {
+        CBlockHeaderAndShortTxIDs block_header_and_short_txids;
+        ds >> block_header_and_short_txids;
+    } catch (const std::ios_base::failure&) {
+    }
+#elif EXT_KEY_DESERIALIZE
+    try {
+        CExtKey ext_key;
+        ds >> ext_key;
+    } catch (const std::ios_base::failure&) {
+    }
+#elif EXT_PUB_KEY_DESERIALIZE
+    try {
+        CExtPubKey ext_pub_key;
+        ds >> ext_pub_key;
+    } catch (const std::ios_base::failure&) {
+    }
+#elif FEE_RATE_DESERIALIZE
+    try {
+        CFeeRate fee_rate;
+        ds >> fee_rate;
+    } catch (const std::ios_base::failure&) {
+    }
+#elif MERKLE_BLOCK_DESERIALIZE
+    try {
+        CMerkleBlock merkle_block;
+        ds >> merkle_block;
+    } catch (const std::ios_base::failure&) {
+    }
+#elif MUTABLE_TRANSACTION_DESERIALIZE
+    try {
+        CMutableTransaction mutable_transaction;
```

Comment:
Couldn't this be put in the `transaction` fuzzer? Both `CMutableTransaction` and `CTransaction` are (de)serialized in the same way

Reply:
So, no need to duplicate seeds.

Reply:
Very good point!

Added commit:

```
Make CTransaction and CMutableTransaction share the same corpus.
Verify that they are (de)serialized in the same way.
```

Please re-review :)